### PR TITLE
feat：update the Wallet name from bitget to Bitget Wallet

### PR DESCRIPTION
**Pull request by kisenyang**
Hi Blocknative team,

This PR update the Wallet name from bitget to Bitget Wallet

Cheers,
bitget wallet team

### CODE REVIEW

File: packages/bitget-wallet/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@web3-onboard/bitget",
+  "name": "@web3-onboard/bitget-wallet",
```
Comment (Adamj1232) [inline packages/bitget-wallet/package.json:2]:
```suggestion
  "name": "@web3-onboard/bitget",
```

File: docs/src/routes/docs/[...4]wallets/[...2]bitget/+page.md [hunk 2]
```diff
@@ -14,14 +14,14 @@ Bitget Wallet SDK wallet module for connecting to Web3-Onboard. Web3-Onboard mak
 <TabPanel value="yarn">
 
 ```sh copy
-yarn add @web3-onboard/bitget
+yarn add @web3-onboard/bitget-wallet
```
Comment (Adamj1232) [inline docs/src/routes/docs/[...4]wallets/[...2]bitget/+page.md:17]:
```suggestion
yarn add @web3-onboard/bitget
```

File: docs/src/routes/docs/[...4]wallets/[...2]bitget/+page.md [hunk 3]
```diff
@@ -14,14 +14,14 @@ Bitget Wallet SDK wallet module for connecting to Web3-Onboard. Web3-Onboard mak
...
 ```
 
   </TabPanel>
   <TabPanel value="npm">
 
 ```sh copy
-npm install @web3-onboard/bitget
+npm install @web3-onboard/bitget-wallet
```
Comment (Adamj1232) [inline docs/src/routes/docs/[...4]wallets/[...2]bitget/+page.md:24]:
```suggestion
npm install @web3-onboard/bitget
```

File: packages/bitget/src/index.ts [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
 }
 
-function bitget(): WalletInit {
+function bitgetWallet(): WalletInit {
   if (typeof window === 'undefined') return () => null
 
   return () => {
     return {
-      label: 'Bitget',
+      label: 'Bitget Wallet',
```
Comment (Adamj1232) [inline packages/bitget/src/index.ts:25]: This line will update the client naming of the wallet.
You can probably revert the changes to add `Wallet` to variable names as it will be implied

Comment (kisenyang) [reply packages/bitget/src/index.ts:25]: I see. Thanks for reply

**Review by Adamj1232 — APPROVED** [review]
Thanks for the PR and collaboration!
